babel/src/test/resources/sql/redshift.iq

```diff
@@ -1843,9 +1843,9 @@ SELECT "REGEXP_REPLACE"('DonecFri@semperpretiumneque.com', '@.*\\.(org|gov|com)$
 
 # REGEXP_SUBSTR ( source_string, pattern [, position [, occurrence
 # [, parameters ] ] ] )
-select regexp_substr('Suspendisse.tristique@nonnisiAenean.edu','@[^.]*');
-SELECT "REGEXP_SUBSTR"('Suspendisse.tristique@nonnisiAenean.edu', '@[^.]*')
-!explain-validated-on calcite
+#select regexp_substr('Suspendisse.tristique@nonnisiAenean.edu','@[^.]*');
+#SELECT REGEXP_SUBSTR('Suspendisse.tristique@nonnisiAenean.edu', '@[^.]*')
+#!explain-validated-on calcite
 
```

Comment:
Why to comment out these tests ?

Reply:
From JIRA seems like it's commented because this change breaks redshift.iq, which should be fixed.

Reply:
As @amaliujia said, my change is breaking this testcase. Since I haven't explored calcite-babel or redshift.iq particularly, I am not sure how it is supposed to be. From what I observed and understood, the expected translated query was "REGEXP_SUBSTR" with double quotes until it wasn't handled. After I added it in to the standard, the double quotes were removed and the test case assertion failed. 
Please let me know a proper fix for this.